Analyzer/include/Histo.h

```diff
@@ -118,7 +120,7 @@ class Histo1D : public Histo_FirstChild<TH1>
             else if(type.find("char")           != std::string::npos) vecFill<char>        ( weight, tr );
             else if(type.find("short")          != std::string::npos) vecFill<short>       ( weight, tr );
             else if(type.find("long")           != std::string::npos) vecFill<long>        ( weight, tr );        
-            else if(type.find("TLorentzVector") != std::string::npos) vecFilltlv           ( weight, tr );        
+            else if(type.find("utility::LorentzVector") != std::string::npos) vecFilltlv           ( weight, tr );        
```

Comment:
Does this work? I wasn't sure if the fundamental name of the type changes with the typedef.

Reply:
Also, we shouldn't remove the TLorentzVector option from this file.

Reply:
I will put back

Reply:
Done in ad24937